.gitignore

```diff
@@ -1 +1,2 @@
 .vagrant/
+.idea/
```

Comment:
This is particular to your environment (intellij idea?), shouldn't go into a shared piece. 
For particular cases you can ignore the file by adding an entry to the `.git/info/exclude`
here is an explanation http://stackoverflow.com/questions/22906851/when-would-you-use-git-info-exclude-instead-of-gitignore-to-exclude-files